src/aks-preview/HISTORY.rst

```diff
@@ -22,6 +22,13 @@ Pending
 +++++++
 * Ignore invalid ip error for `--api-server-authorized-ip-ranges`.
 
+3.0.0b8
++++++++
```

Comment:
should move these to pending or choose a new version

Reply:
thanks, fixed